kedro/framework/cli/project.py

```diff
@@ -374,18 +398,21 @@ def activate_nbstripout(
     help=PARAMS_ARG_HELP,
     callback=_split_params,
 )
-# pylint: disable=too-many-arguments,unused-argument
+# pylint: disable=too-many-arguments,unused-argument, too-many-locals
 def run(
     tag,
+    tags,
     env,
     runner,
     is_async,
     node_names,
+    nodes_names,
```

Comment:
This is named this way for differentiation from the already existing `node_names` - in #2245 I'd suggest keeping `node_names` as the parameter name

Reply:
Fine to keep `node_names` for now in #2245 but as part of https://github.com/kedro-org/kedro/issues/2247 I think we should consider just calling this `nodes`. (Just while I remember, Ivan's argument for why we might not want to do this: it's deliberately written as `node_names` to deter people from trying to pass node objects themselves to the underlying `session.run`. My argument for it is consistency and simplicity, particularly when we eventually return to https://github.com/kedro-org/kedro/pull/1423)